docs/user/experimental/keyboard/modal.md

```diff
@@ -86,7 +80,8 @@ To bind keys that do not have modifiers (eg just pressing `a`), add entries like
     "ac": "copyToClipboard",
     "ax": "cutToClipboard",
     "ap": "pasteFromClipboard",
-    "ad": "followLink"
+    "ad": "followLink",
+    "aw": "remove",
```

Comment:
aw is already wrap, i thought.

aa? as in "aaaaaah it's horrible delete it quick"?

edit: i see you stole "x" below. that seems good. eXcise.

Reply:
unfortunate to have this info duplicated here. maybe for the moment just link to the fixture file?

Reply:
Oh right, duh, I forgot we were in dueling keyboard mode. Ship it.